basket/settings.py

```diff
@@ -85,26 +88,27 @@ def path(*args):
 default_email_backend = "django.core.mail.backends.console.EmailBackend" if DEBUG else "django.core.mail.backends.smtp.EmailBackend"
 EMAIL_BACKEND = config("EMAIL_BACKEND", default=default_email_backend)
 EMAIL_HOST = config("EMAIL_HOST", default="localhost")
-EMAIL_PORT = config("EMAIL_PORT", default=25, cast=int)
-EMAIL_USE_TLS = config("EMAIL_USE_TLS", default=False, cast=bool)
+EMAIL_PORT = config("EMAIL_PORT", parser=int, default="25")
+EMAIL_USE_TLS = config("EMAIL_USE_TLS", parser=bool, default="false")
 EMAIL_SUBJECT_PREFIX = config("EMAIL_SUBJECT_PREFIX", default="[basket] ")
 EMAIL_HOST_USER = config("EMAIL_HOST_USER", default="")
 EMAIL_HOST_PASSWORD = config("EMAIL_HOST_PASSWORD", default="")
 
 ALLOWED_HOSTS = config(
     "ALLOWED_HOSTS",
-    default=".allizom.org, .moz.works, basket.mozmar.org, basket.mozilla.com, basket.mozilla.org",
-    cast=Csv(),
+    parser=ListOf(str, allow_empty=False),
+    default="basket-dev.allizom.org,basket.allizom.org,basket.mozilla.org",
```

Comment:
Just calling out that these defaults are different, but I can see you've updated them to current vals

Reply:
Yeah we had some old stuff in there